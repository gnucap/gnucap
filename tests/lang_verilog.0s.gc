
verilog

module foo();
	parameter string fs="default";
	parameter string ft={"def" + "ault"};
endmodule;

parameter a=1.;
parameter string s="default";

// BUG still need {} in c_param
parameter string t={"def" + "ault"};

eval s
eval t

foo #(.fs("testvalue")) f();
`op
list

list

spice

.list
