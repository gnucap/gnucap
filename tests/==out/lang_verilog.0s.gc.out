s="default"
t="default"
parameter "testvalue" not specified, using default
parameter "testvalue" value is "NOT_INPUT"
parameter "testvalue" not specified, using default
parameter "testvalue" value is "NOT_INPUT"
parameter "testvalue" not specified, using default
parameter "testvalue" value is "NOT_INPUT"
#          
parameter "testvalue" not specified, using default
parameter "testvalue" value is "NOT_INPUT"
parameter "testvalue" not specified, using default
parameter "testvalue" value is "NOT_INPUT"
 27.       
parameter "testvalue" not specified, using default
parameter "testvalue" value is "NOT_INPUT"
module foo ();
endmodule // foo

// BUG still need {} in c_param 
foo #(.fs("testvalue")) f ();
parameter "testvalue" not specified, using default
parameter "testvalue" value is "NOT_INPUT"
module foo ();
endmodule // foo

// BUG still need {} in c_param 
foo #(.fs("testvalue")) f ();
parameter "testvalue" not specified, using default
parameter "testvalue" value is "NOT_INPUT"
.subckt foo (  )
.ends foo
*// BUG still need {} in c_param 
f (  )  foo  fs="testvalue"
